tests/unit/fixtures/config.yaml

```diff
@@ -2,6 +2,11 @@
 backends:
   cloud_monitoring:
     project_id: ${PROJECT_ID}
+  splunk:
```

Comment:
Not sure this is the best option to define config values. The other backends do not have hard-coded values like these. Let me check how they work under the hood.

Reply:
Given how long ago I wrote this, I'm not sure why I had to add them.
unit tests seem to run fine without. I'll remove this block.